nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -445,3 +446,171 @@ def test_plot_prob_atlas():
                               threshold=0.2)
     # Testing the 4D plot prob atlas with contours
     display = plot_prob_atlas(img, view_type='continuous')
+
+
+def test_get_plot_stat_map_params_pos_neg():
+    # data with positive and negative range
+    affine = np.eye(4)
+    data = np.array([[-.5, 1., 0.],
+                     [0., 1., -.2],
+                     [1.5, 2.5, 3.]])
+    img = nibabel.Nifti1Image(data, affine)
+
+    # symmtric_cbar set to True
+    cbar_vmin, cbar_vmax, vmin, vmax = _get_plot_stat_map_params(img, None,
```

Comment:
Just for clarity, it'd be quite good to name the parameters in the _get_plot_stat_map_params function, i.e.:

```python
_get_plot_stat_map_params(img, vmax=2, symmetric_cbar=True, kwargs={})
```

I find myself always going back to the function signature ...